src/parse_args.ts

```diff
@@ -104,6 +108,11 @@ program
     '--quiet-exit-code',
     'always return 0 as an exit code status, regardless of test pass / fail. Only return > 0 exit codes on internal errors where the suite could not be run'
   )
+  .addOption(
+    new Option('--throttle <throttle>', 'Comma separated list of options to throttle network conditions for download throughput (d) in megabytes/second, upload throughput (u) in megabytes/second and latency (l) in milliseconds. Ex: --throttle "3d,2u,10l"')
```

Comment:
nit: How about `throttling` `no-throttling` for naming? https://github.com/tj/commander.js#other-option-types-negatable-boolean-and-booleanvalue